### DISCUSSION TITLE
Add current Layer to display

### DISCUSSION THREAD

**Discussion by jxer** [Jyers/Marlin#727] [Ideas]
Is it possible to get a layer level? I would actually prefer it over the x/y coordinates at the bottom, as they constantly change and don't provide ME any usful information, could we make it a check box option to maybe display layer 1 of 300 etc, i know the z offset gives a rough idea, but when troubleshooting a failing section of a print i could determine the layer and look at the print in cura to see if its a design flaw or printer issue

**Comment by t3chguy** [comment]
https://plugins.octoprint.org/plugins/DisplayLayerProgress/ can send M117 to Marlin for it to display it on the UI.

**Reply by jxer** [reply]
I do not use octoprint. but thank you. I primarily print so far from lcd screen.

**Reply by t3chguy** [reply]
Get your slicer to add the M117s then.

**Reply by jxer** [reply]
I will do that, still would like to see it baked in.

**Reply by t3chguy** [reply]
Marlin doesn't support that as far as I can tell. Makes sense given your slicer might do things like z hop or dynamic layer heights

**Reply by jxer** [reply]
> Marlin doesn't support that as far as I can tell. Makes sense given your slicer might do things like z hop or dynamic layer heights

I found it post processing for cura. giving it a go shortly. thanks for the direction

**Reply by Cal-LF** [reply]
Let us know the results. Looks simple enough to try.

**Reply by jxer** [reply]
it works fine. Does what I need it too
